### ISSUE TITLE
Support v3

### ISSUE THREAD

**Issue by mokhosh** [ariaieboy/filament-jalali-datetime#1]
Hey @ariaieboy, any plans to support v3?
This is the current state:
```
- ariaieboy/filament-jalali-datetime is locked to version 1.1.1 and an update of this package was not requested.
- ariaieboy/filament-jalali-datetime 1.1.1 requires filament/filament ^2.12 -> found filament/filament[v2.12.0, ..., v2.17.50] but it conflicts with your root composer.json require (^3.0).
```

**Comment by ariaieboy** [comment]
I am working on it.
The new version plugin system changed a lot, and I need to read the latest documentation.

**Comment by mokhosh** [comment]
Thanks, but I would say just change the required version and drop it in a v3 installation and see if it works.
Shouldn't be anything complicated since this is a very light plugin.

**Comment by ariaieboy** [comment]
Feel free to make a PR.

**Comment by ariaieboy** [comment]
Fixed in [v1.2.0](https://github.com/ariaieboy/filament-jalali-datetime/releases/tag/1.2.0)
